newrelic-alerts-configurator/src/test/java/com/ocadotechnology/newrelic/alertsconfigurator/AbstractConfiguratorTest.java

```diff
@@ -1,20 +1,27 @@
 package com.ocadotechnology.newrelic.alertsconfigurator;
 
-import com.ocadotechnology.newrelic.apiclient.*;
+import com.ocadotechnology.newrelic.apiclient.AlertsChannelsApi;
+import com.ocadotechnology.newrelic.apiclient.AlertsPoliciesApi;
+import com.ocadotechnology.newrelic.apiclient.ApplicationsApi;
+import com.ocadotechnology.newrelic.apiclient.DashboardsApi;
+import com.ocadotechnology.newrelic.apiclient.KeyTransactionsApi;
+import com.ocadotechnology.newrelic.apiclient.NewRelicApi;
+import com.ocadotechnology.newrelic.apiclient.PolicyItemApi;
+import com.ocadotechnology.newrelic.apiclient.ServersApi;
+import com.ocadotechnology.newrelic.apiclient.UsersApi;
 import com.ocadotechnology.newrelic.apiclient.model.conditions.AlertsCondition;
-import com.ocadotechnology.newrelic.apiclient.model.conditions.nrql.AlertsNrqlCondition;
 import com.ocadotechnology.newrelic.apiclient.model.conditions.external.AlertsExternalServiceCondition;
+import com.ocadotechnology.newrelic.apiclient.model.conditions.nrql.AlertsNrqlCondition;
 import org.junit.Before;
-import org.junit.Rule;
+import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnit;
-import org.mockito.junit.MockitoRule;
+import org.mockito.runners.MockitoJUnitRunner;
 
 import static org.mockito.Mockito.when;
 
+@RunWith(MockitoJUnitRunner.class)
 public abstract class AbstractConfiguratorTest {
-    @Rule
-    public final MockitoRule mockito = MockitoJUnit.rule();
```

Comment:
This may not be explicitely said in our Contribution guide, but we prefer to use rules over runners, and we do so in other tests :)